modules/k8s-cluster/README.md

```diff
@@ -0,0 +1,54 @@
+# k8s-cluster
```

Comment:
please have some description here about the module,
can we have this as top level module for this repo similar that we have for aws k8s module?
will be good to follow same structure in module structure for k8s setups in different clouds.